add support for multiple selections

benoit-pierre opened the pull request:
The selection elements are joined by newlines (line mode), or spaces (other modes). To ensure the pasted text is not interpreted, `tmux paste-buffer` bracketed mode is now used.

Close #79.

laktak commented:
Thank you for your PR! :+1: 

How do you actually select the items as `tab` is already used to insert by default? Shift-Tab? I think we should also mention this in the documentation/help.

And IMO it would be better to separate the selections with `\n` in 'all' mode like you already do for line mode.

benoit-pierre commented:
> How do you actually select the items as `tab` is already used to insert by default? Shift-Tab? I think we should also mention this in the documentation/help.

Yes, I updated the help. I also fixed the formatting to please `shfmt`.

> And IMO it would be better to separate the selections with `\n` in 'all' mode like you already do for line mode.

I think space-separated words makes more sense / is more convenient.

sergeylukin commented:
WP! One note though, I had to remap insert key to get this working, so `tmux.conf`:

```
set -g @extrakto_insert_key 'ctrl-i'
```

Cheers!

dvdbng commented:
Love it! Could you add the binding to the header (line 140)? something like `shift-tab=select multiple`.

dvdbng commented:
> I think space-separated words makes more sense / is more convenient.

+1, for my use cases (hg add-ing a bunch of files after hg status spaces make more sense

dufferzafar commented:
> I had to remap insert key and restart tmux (`tmux kill-server`) to get this working, so `tmux.conf`

I did not have to do this. Just copying the extrakto.sh script to `~/.tmux/plugins/extrakto/scripts/extrakto.sh` worked.

@laktak Any reason this PR isn't merged yet? It seems to be working fine for users.

laktak commented:
@dufferzafar sorry, there was not enough feedback at first and then I was very busy.

@benoit-pierre Thanks again for your PR!
